### ISSUE TITLE
Cannot redeploy chequebook

### ISSUE THREAD

**Issue by cobordism** [ethersphere/bee#1518]
#### Summary
My swarm node apparently has a chequebook deployed using the wrong key.
The chequebook is using the key held by swarm wheres all other actions are taken by the key in clef, including node id.

#### Steps to reproduce
```
# sudo -u bee bee --config /etc/bee/bee.yaml deploy
INFO[2021-03-31T16:30:11+02:00] using swarm network address through clef: 7cf1b3f28c545abc8b1a25de17b495566b12f1c77e848a6977cedcb67c7b4e43 
INFO[2021-03-31T16:30:11+02:00] swarm public key 03ab873c7a963790bf87e67645f05d466115350c94d79287a71e0c336133413f97 
DEBU[2021-03-31T16:30:11+02:00] using existing libp2p key                    
DEBU[2021-03-31T16:30:11+02:00] using existing pss key                       
INFO[2021-03-31T16:30:11+02:00] pss public key 033625c2a3a6f86368fdd3c948d330329513b0a1e33ee1dd7008ed41930fe26180 
INFO[2021-03-31T16:30:11+02:00] using ethereum address ddf059c4c590b82a6397c4208e465829c88f7f29 
[2021-03-31T16:30:13.0793974+02:00]  INFO using default factory address for chain id 5: f0277caffea72734853b834afc9892461ea18474
INFO[2021-03-31T16:30:13+02:00] using existing chequebook f6e035cb6916d5f40e6ec7d35c9442a485daeef4 
```
That chequebook you see there does not belong to the key that determines the network address. 

#### Expected behavior
I expect `bee deploy` to always deploy a correct chequebook if none is present.

**Comment by Eknir** [comment]
Before 0.5.3 it was possible to start the node with one overlay address and then next time start with another. This could lead to many edge-conditions, one of which you have experienced. This was fixed with (#1377). 

`bee deploy` is a command intended to run by a user who does not have a chequebook. Since this is not your situation, there is nothing to change here.

Essentially, your bee has a corrupted state and the best way out of this is a clean install. If you can cashout all your cheques, I would advice to do that and start with a clean state.

**Comment by cobordism** [comment]
I'll just wait for 1.0
